cmudb/extensions/hutch/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Set up the TScout extension.
```

Comment:
I'm under the impression that we are no longer using cmake?
If so, do we still need these files?

(@lmwnshn)

Reply:
No idea what happened here. 
I re-merged the upstream cmu-db/postgres and this sorted itself out.